### PR TITLE
added FAQ section with mesh cell count question

### PR DESCRIPTION
**Pull request by anilyil**
## Purpose
This PR adds a FAQ section to the tutorial docs. I also added an answer to the common question on cell counts in a mesh.

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

### CODE REVIEW

File: machAeroTutorials/faq.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    Total Zones: 9
+    Total Cells: 193536
+    Total Nodes: 217413
+    Wall Boundary Cells: 4032
+    Wall Boundary Nodes: 4437
+
+
+Running ADflow with a mesh will also print the number of cells to the terminal.
+For example, after printing the full options dictionary, ADflow will print:
+::
```
Comment (eirikurj) [inline machAeroTutorials/faq.rst:32]: Same here

File: machAeroTutorials/faq.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    # Grid level: 1, Total number of cells: 193536
+    #
+
+
+Finally, users should note that for overset meshes, the number of "compute" cells will be different than number of total cells.
+In both methods above, the code will print the total number of cells present in the CGNS file.
+However, with overset grids, a portion of the cells will be "blanked".
+During the overset hole cutting, ADflow will print some detailed information about this process.
+For example:
+::
```
Comment (eirikurj) [inline machAeroTutorials/faq.rst:44]: Same here

File: machAeroTutorials/faq.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    | Fringe Cells            :       46574
+    | Blanked Cells           :       15135
+    | Explicitly Blanked Cells:           0
+    | Flooded   Cells         :        2458
+    | FloodSeed Cells         :        5161
+    +--------------------------------+
+    Total number of orphans:           0
+
+The last iteration of the hole cutting algorithm will print the number of ``Compute Cells``.
+With simulations using overset meshes, the code will still loop over all of the cells during residual computations; however, the blanked cells do not add directly to the cost of linear solutions with implicit solvers or the adjoint solver. As a result, the residual evaluations' cost will still be proportional to the total number of cells, while the cost of linear solutions will roughly be proportional to the number of compute cells. See the :ref:`overset theory guide <overset_theory>` for more details.
```
Comment (eirikurj) [inline machAeroTutorials/faq.rst:101]: Please to one line per sentence.

**Review by eirikurj — CHANGES_REQUESTED** [review]
Looks good, very minimal comments

File: machAeroTutorials/faq.rst [hunk 4]
```diff
@@ -0,0 +1,101 @@
+.. _faq:
+
+*******************************************************
+Frequently Asked Questions
+*******************************************************
```
[the remark this answers is not in the document]
Comment (anilyil) [reply machAeroTutorials/faq.rst:5]: done

File: machAeroTutorials/faq.rst [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+In this section, we have a list of frequently asked questions and answers.
+
+How do I obtain the cell count in a mesh?
+-----------------------------------------
+
+There are severak ways to get the cell count information from a mesh file.
```
[the remark this answers is not in the document]
Comment (anilyil) [reply machAeroTutorials/faq.rst:12]: done

File: machAeroTutorials/faq.rst [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+The simplest way is to use the ``info`` command in CGNS Utilities will print this information quickly to the terminal.
+
+::
```
[the remark this answers is not in the document]
Comment (anilyil) [reply machAeroTutorials/faq.rst:16]: done

File: machAeroTutorials/faq.rst [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+    MACH-Aero/tutorial/aero/analysis $ cgns_utils info wing_vol.cgns
+
+The output should look like:
+::
```
[the remark this answers is not in the document]
Comment (anilyil) [reply machAeroTutorials/faq.rst:21]: done

File: machAeroTutorials/faq.rst (hunk 1 above)
Comment (anilyil) [reply machAeroTutorials/faq.rst:32]: done

File: machAeroTutorials/faq.rst (hunk 2 above)
Comment (anilyil) [reply machAeroTutorials/faq.rst:44]: done

File: machAeroTutorials/faq.rst (hunk 3 above)
Comment (anilyil) [reply machAeroTutorials/faq.rst:101]: done

**Comment by anilyil** [comment]
Thanks for the quick review @eirikurj. I addressed your comments.

**Review by eirikurj — APPROVED** [review]

**Review by nwu63 — APPROVED** [review]

**Review by marcomangano — APPROVED** [review]

**Pull request merged**